docs/api/synchronization_library/barrier.md

```diff
@@ -33,8 +45,9 @@ __device__ std::uint64_t* barrier_native_handle(
 
 }}
 ```
-* Expects: `b` is in shared memory;
-* Returns: a pointer to the PTX "mbarrier" subobject of the `barrier` object.
+
+- Expects: `b` is in `__shared__` memory.
+- Returns: a pointer to the PTX "mbarrier" subobject of the `barrier` object.
 
```

Comment:
"Shared memory" is a phrase with meaning though, right? I think I prefer prose here instead of code font "`__shared__`".

Reply:
I don't want to say "shared memory" because people who are NOT familiar with CUDA will read that as... well... the traditional meaning of shared memory: e.g. interprocess shared memory.